### ISSUE TITLE
Skeleton

### ISSUE THREAD

**Issue by bulygin1985** [bulygin1985/object_detection#5]
1) train.py for 
                     a) overfit on several images of Pascal VOC
                     b) train on Pascal VOC train
2)  Examples:
overfit
python train.py --overfit
a) default filename to save model
b) fixed lr_schedule with Adam, lr =0 .01, lr=0.001

3) train:
python train.py
Train on PascalVoc train

Purpose: overfit to loss < 1.0

**Comment by bubanoid** [comment]
At what condition should we finish the loop over the epochs when the `--overfit` parameter wasn't passed?
